### ISSUE TITLE
Focused Weapon die damage incorrect in some cases

### ISSUE THREAD

**Issue by mjc1677** [Vek17/WrathMods-TabletopTweaks#510]
I've located an example where the Focused Weapon option of the Advanced Weapon Feat doesn't appear to be calculating properly.

Just something I noticed, so not sure where else this bug might be occurring, but currently I had Regill take the Advanced Weapon Training Feat with the Focused Weapon choice at he's level 15 using the +2 axiomatic gnome hooked hammer.
The damage should be, per everything I've managed to find regarding how Focused Weapon and Sacred Weapon works, 1d10/1d10, but it's coming out to 1d6/1d6 as if he were medium level 1-4. Additionally, the sacred weapon dice is overwriting the weapon damage dice when the weapon is higher. Finally, the die reduction due to Small size is being applied at the end so the damage die were finally calculated at 1d4/1d4.
I've included an example screenshot of the log:

![Focused_weapon_bug](https://user-images.githubusercontent.com/8923317/196738510-f33e4372-ca6d-469b-aab6-5169d9eb83b5.png)

**Comment by Vek17** [comment]
Is he fighter level 15 or just 15? Die scale is based on fighter level

**Comment by mjclaussen** [comment]
5 Fighter (Armiger) 10 Hellknight, so that follows as to why he's getting 1d6/1d6 for focused weapon damage.

So I suppose the issue is 
1: the 1d6 focused weapon damage is being selected over the 1d8 regular weapon damage
2: it should be 1d6 after adjustment for small size. Basically the size reduction for damage dice is being applied twice.

Might just be easiest to give the character medium size sacred weapon damage and let the game apply the size adjustment as needed, but dunno if that might cause other issues.

**Comment by Vek17** [comment]
If you want to use the regular damGe simply disable the focused weapon
toggle

**Comment by mjclaussen** [comment]
Didn't realize that was an option, thanks!
So that just leaves 2 as the only issue.

**Comment by Vek17** [comment]
2 isn't an issue, focused weapon applies before size changes so if it
shifts to 1d6 size can down shift it to 1d4

**Comment by mjclaussen** [comment]
Right, but for a level 5 fighter with focused weapon should be the same as a level 5 warpriest sacred weapon, which should be 1d8, which would then go to 1d6 for small size? In the example I gave it's correctly 1d6 but then goes down to 1d4.
I'm going off the d20pfsrd here:
https://www.d20pfsrd.com/classes/hybrid-classes/warpriest/
https://www.d20pfsrd.com/classes/core-classes/fighter/#TOC-Advanced-Weapon-Training

**Comment by Fiddydamage** [comment]
> Right, but for a level 5 fighter with focused weapon should be the same as a level 5 warpriest sacred weapon, which should be 1d8, which would then go to 1d6 for small size? In the example I gave it's correctly 1d6 but then goes down to 1d4. I'm going off the d20pfsrd here: https://www.d20pfsrd.com/classes/hybrid-classes/warpriest/ https://www.d20pfsrd.com/classes/core-classes/fighter/#TOC-Advanced-Weapon-Training

Regill is a gnome. Gnomes are small creatures, not medium. According to the pfsrd site you linked, sacred weapon for small creatures are 1d6 at 5th level. The damage listed in the table near the top is for medium creatures, you need to scroll down to see the table for small/large ones.

Seems working as intended.
